### PR TITLE
Removes Mortar Recipe's cement recipe book requirement

### LINKED ISSUE #74015: Mortar recipe is too hard to get
### Is your feature request related to a problem? Please describe.

Currently the only source for the Mortar recipe is in Concrete Constructions, a pretty uncommon book.

In reality, the recipe is usually printed on the back of the bags of cement needed to make it.

### Solution you would like.

The knowledge of how to make mortar from cement is simple, widespread, and usually found on bags of cement. The recipe should probably be autolearn.

### Describe alternatives you have considered.

Make it possible for containers to have recipes printed on them.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by VileReviled**
#### Summary

Balance "Mortar is now an autolearned recipe"

#### Purpose of change

Fixes #74015.

#### Describe the solution

Makes Mortar into an autolearned recipe, and removes book requirement. 

#### Describe alternatives you've considered

As many commenters in #74015 agree, simply making this into an autolearned recipe is a little pathetic. I'd love to revisit this once I have more experience

#### Testing

Linting produced no errors I can tell. I can't test through Docker currently; testing welcome!!!

#### Additional context

This is my first PR! Feedback appreciated

### CODE REVIEW

**Comment by VileReviled** [comment]
> This would be an ideal way to resolve this. If you want me to I can update your PR with this and fix the itemgroups that would need updating as well.

Yes please! I agree that this is ideal.

File: data/json/recipes/other/parts_construction.json [hunk 1]
```diff
@@ -664,8 +664,7 @@
     "skill_used": "fabrication",
     "difficulty": 2,
     "time": "10 m",
-    "autolearn": false,
-    "book_learn": [ [ "concrete_book", 2 ] ],
+    "autolearn": true,
```
Comment (Maleclypse) [inline data/json/recipes/other/parts_construction.json:667]:
```suggestion
    "autolearn": false,
    "book_learn": [ [ "concrete_book", 2 ] ],
```

**Comment by Maleclypse** [comment]
I really appreciate you putting a PR together for this.  Thank you for doing this and I hope to see you contribute some more :)

File: data/json/items/containers/containers.json [hunk 2]
```diff
@@ -146,6 +146,29 @@
...
+      "effect_on_conditions": [ "EOC_CEMENT_INSTRUCTIONS" ]
+    }
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_CEMENT_INSTRUCTIONS",
+    "condition": { "and": [ { "u_has_trait": "ILLITERATE" }, { "not": "u_driving" } ] },
+    "effect": [
+      { "u_learn_recipe": "mortar_build" }
+    ]
```
[the remark this answers is not in the document]
Comment (VileReviled) [reply data/json/items/containers/containers.json:170]: Similar code in the same file also only spans one line, so I think it's best to commit this. Anyone confirm?

[the remark this answers is not in the document]
Comment (Maleclypse) [reply data/json/items/containers/containers.json:170]: That's correct.

File: data/json/items/containers/containers.json [hunk 3]
```diff
@@ -146,6 +146,27 @@
     "color": "white",
     "flags": [ "COLLAPSE_CONTENTS" ]
   },
+  {
+    "id": "bag_durasack_cement",
+    "type": "GENERIC",
+    "category": "container",
```
Comment (Maleclypse) [inline data/json/items/containers/containers.json:152]:
```suggestion
    "category": "container",
```

**Pull request merged**